src/Aacotroneo/Saml2/Http/Controllers/Saml2Controller.php

```diff
@@ -131,7 +130,7 @@ private function getHL7MessageFromRequest( $user )
      */
     private function getUserFromRequest( $samlUser )
```

Comment:
Can we move this method to HL7 helper? We should try to have as little as possible on the vendor. Also please update comments to reflect we are not only checking by  email